website/docs/proposals/20250630-kaito-cli.md

```diff
@@ -0,0 +1,526 @@
+  --config my-lora-config \
+  --input pvc://training-data \
+  --output pvc://output-adapters \
+  --epochs 5 \
+  --batch-size 4
+```
+
+#### Tuning Management
+
+```bash
+# Tuning status and monitoring
+kubectl kaito tune status my-tuning
+kubectl kaito tune logs my-tuning --follow
+kubectl kaito tune metrics my-tuning --show-gpu-utilization
+
+# Tuning with resource optimization
+kubectl kaito tune --name my-tuning --model llama-3.3-70b --method qlora \
+  --auto-optimize-resources \
+  --instance-type Standard_NC24ads_A100_v4 \
+  --show-cost-estimate
+```
+
+### RAG Operations
+
+#### Deployment
+
+```bash
+# Deploy RAG engine with local embedding
+kubectl kaito rag deploy --name my-rag --embedding-model BAAI/bge-small-en-v1.5 --inference-url http://my-service/v1/completions
+
+# Deploy complete RAG stack with workspace integration
+kubectl kaito rag deploy --name my-rag-app \
+  --embedding-model BAAI/bge-small-en-v1.5 \
+  --llm-workspace my-inference-workspace \
+  --vector-store faiss \
+  --instance-type Standard_NC6s_v3
+
+# Deploy RAG with external inference service
+kubectl kaito rag deploy --name my-rag-external \
+  --embedding-model BAAI/bge-small-en-v1.5 \
+  --inference-url https://api-inference.huggingface.co/models/HuggingFaceH4/zephyr-7b-beta/v1/completions \
+  --inference-secret huggingface-token
+```
+
+#### Document and Query Management
+
+```bash
+# Manage RAG indices and documents
+kubectl kaito rag index --name my-rag --add-document doc.pdf
+kubectl kaito rag index --name my-rag --add-documents-dir ./docs --batch-size 10
+kubectl kaito rag index --name my-rag --list-documents
+kubectl kaito rag index --name my-rag --delete-document doc-id-123
+
+# Query and test RAG system
+kubectl kaito rag query --name my-rag --question "What is KAITO?"
+kubectl kaito rag query --name my-rag --question "What is KAITO?" --show-context --top-k 3
+kubectl kaito rag test --name my-rag --test-file questions.csv --output results.json
+```
+
+#### RAG System Management
+
+```bash
+# RAG system management
+kubectl kaito rag status my-rag
+kubectl kaito rag logs my-rag --follow
+kubectl kaito rag scale my-rag --replicas 3
+kubectl kaito rag backup my-rag --output ./backup/
+kubectl kaito rag restore my-rag --from ./backup/
+```
+
+### Model Management
+
+#### Model Catalog Operations
+
+```bash
+# List supported models with detailed information
+kubectl kaito models list --output table
+kubectl kaito models list --output json
+
+# Show model details
+kubectl kaito models describe phi-3.5-mini-instruct
+kubectl kaito models describe llama-3.3-70b-instruct --show-runtime-params
+
+# Model compatibility and validation
+kubectl kaito models validate --model phi-3-mini --instance-type Standard_NC6s_v3
+kubectl kaito models validate --model llama-3.3-70b --distributed --nodes 4
+
+# Runtime selection and configuration
+kubectl kaito models runtime --model phi-3-mini --runtime vllm
```

Comment:
What do you mean by runtime selection? I thought the user would select the runtime (vllm/transformers) in `kubectl kaito deploy` command